src/invite/Invite.hooks.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { noop } from 'lodash'
+import { useEffect, useState } from 'react'
+import { Share } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+
+import { InviteEvents } from 'src/analytics/Events'
+import { useShareUrl } from './Invite.hooks'
+
+jest.mock('react-native', () => ({
+  Share: {
+    share: jest.fn(),
+  },
+}))
+
+jest.mock('src/analytics/ValoraAnalytics', () => ({
+  track: jest.fn(),
+}))
+
+jest.mock('react-i18next', () => ({
+  useTranslation: jest.fn().mockReturnValue({
+    t: (name: string) => name,
+  }),
+}))
+
+jest.mock('react', () => ({
+  useState: jest.fn().mockReturnValue([null, () => null]),
+  useEffect: jest.fn().mockImplementation((cb) => cb()),
```

Comment:
I'm open to suggestions. Testing hooks is always a little weird, mocking React hooks seems to give me the best results. I only mock React in the hooks test file, so testing the render of the `Invite` component still uses non-mocked React.

Reply:
+1 to what @kathaypacific said.
Can we test the hook inside a dummy comment like we did [here](https://github.com/valora-inc/wallet/blob/2b28b691640c52bdeeb1b49a7751c5bdf3b764fe/src/tokens/hooks.test.tsx) instead?

Reply:
@kathaypacific @jeanregisser let me know how you feel about my solution. I tested both the hook and the component in one file. I ended up mocking the utils without testing, since there's very little to test there, and I ran into some troubles mocking DynamicLinks properly. I'm happy to take another pass if you feel we're lacking coverage anywhere.